examples/explorer-template/packages/client/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "client",
+  "version": "0.0.0",
+  "private": true,
+  "license": "MIT",
+  "type": "module",
+  "scripts": {
+    "build": "vite build",
+    "dev": "wait-port localhost:8545 && vite",
+    "preview": "vite preview",
+    "test": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@latticexyz/common": "2.1.0",
+    "@latticexyz/dev-tools": "2.1.0",
+    "@latticexyz/react": "2.1.0",
+    "@latticexyz/recs": "2.1.0",
+    "@latticexyz/schema-type": "2.1.0",
+    "@latticexyz/store-sync": "2.1.0",
+    "@latticexyz/utils": "2.1.0",
+    "@latticexyz/world": "2.1.0",
```

Comment:
we're going to have version issues if we have some MUD packages coming from npm and some linked locally

we should change all these to `link:../../packages/*`